pages/perfherder.py

```diff
@@ -13,7 +13,6 @@ class PerfherderPage(Base):
 
     def wait_for_page_to_load(self):
         self.wait.until(lambda s: self.is_graph_chooser_displayed)
-        return self
```

Comment:
Thanks Dave! Because I was a little bit confused about that.
@rbillings here is the answer for my yesterdays question :)